src/Storage/Storage.Management/ChangeLog.md

```diff
@@ -18,6 +18,9 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Added warning messages for an upcoming breaking change for creating and updating a Queue access policy
+    - `New-AzStorageQueueStoredAccessPolicy`
+    - `Set-AzStorageQueueStoredAccessPolicy`
```

Comment:
Hi @yifanz7 , could you provide more details in change log, like what kind of breaking change will happen in the future.

Reply:
Updated. Please let me know if more details are needed